### PR TITLE
raise PR on config change

### PR DESCRIPTION
**Pull request by Manali-Jain-squareops**
## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: .github/workflows/config-change-0helm-pr.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      - staging
+    types:
+      - closed
+    paths:
+      - docker.local/config/0chain.yaml
+      - docker.local/config/sc.yaml
+   
+jobs:
+  ConfigChange:
+    runs-on: docker-builds
```
Comment (shahnawaz-creator) [inline .github/workflows/config-change-0helm-pr.yaml]: use `arc-runner` here.

File: .github/workflows/config-change-0helm-pr.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          publish_dir: ./0helm/values/activeset/common
+          destination_dir: ./0helm/values/activeset/common
+          publish_branch: one-pool-staging-test
+          keep_files: true
+          external_repository: "0chain/0helm"
+          user_name: "github-actions[bot]"
+          user_email: "github-actions[bot]@users.noreply.github.com"
+
+  PR:
+    runs-on: 'docker-builds'
```
Comment (shahnawaz-creator) [inline .github/workflows/config-change-0helm-pr.yaml]: replace it with `arc-runner`

File: .github/workflows/config-change-0helm-pr.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    environment: 'HelmRepo'
+    needs: [ConfigChange]
+    steps:
+      - name: Raise PR
+        uses: devops-infra/action-pull-request@v0.5.5
+        with:
+          github_token: ${{ secrets.ACCESS_TOKEN }}
+          source_branch: one-pool-staging-test
+          target_branch: one-pool-staging
```
Comment (shahnawaz-creator) [inline .github/workflows/config-change-0helm-pr.yaml]: raise the PR wrt to development branch.
Create new folder `updated configs` in 0helm repo on development branch. Add sc.yaml , 0chain.yaml, blobber.yaml, validator.yaml & 0box.yaml config file inside it and always raise the PR wrt to development branch.

File: .github/workflows/config-change-0helm-pr.yaml (hunk 1 above)
Comment (Manali-Jain-squareops) [reply .github/workflows/config-change-0helm-pr.yaml]: added

File: .github/workflows/config-change-0helm-pr.yaml (hunk 2 above)
Comment (Manali-Jain-squareops) [reply .github/workflows/config-change-0helm-pr.yaml]: added

File: .github/workflows/config-change-0helm-pr.yaml (hunk 3 above)
Comment (Manali-Jain-squareops) [reply .github/workflows/config-change-0helm-pr.yaml]: updated

**Review by shahnawaz-creator — APPROVED** [review]
LGTM

**Review by dabasov — APPROVED** [review]
lgtm

**Pull request merged**